tasks/main.yml

```diff
@@ -71,9 +71,14 @@
   service: name=supervisor state=restarted
   when: supervisord_conf|changed
 
+- name: store path to github key
+  set_fact:
+    github_key_path: "{{ ssh_dir }}/github"
+  when: github_deploy_key|default('')
```

Comment:
Actually do I need that `default` filter?

Reply:
I think so.  If I remember correctly, the `when` clause will throw an error if the variable is not defined without it.